pallets/schemas/src/serde.rs

```diff
@@ -22,72 +17,81 @@ pub fn validate_json_model(json_schema: Vec<u8>) -> Result<(), SerdeError> {
 	}
 }
 
-#[allow(dead_code)]
-fn create_schema_vec(from_string: &str) -> Vec<u8> {
-	Vec::from(from_string.as_bytes())
-}
+#[cfg(test)]
+mod tests {
+
+	use super::{super::mock::*, *};
+	use frame_support::assert_noop;
+
+	fn create_schema_vec(from_string: &str) -> Vec<u8> {
+		Vec::from(from_string.as_bytes())
+	}
 
-#[test]
```

Comment:
These tests did not change, they are just better wrapped in a `mod tests` to remove dead_code